org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/pom.xml

```diff
@@ -23,6 +23,13 @@
     <artifactId>jdt.ls.extension.core</artifactId>
     <packaging>eclipse-plugin</packaging>
     <name>Che Ls Jdt :: Extension :: Core</name>
+    <dependencies>
```

Comment:
Can you explain why this fixes the "class not found" you encountered? And shouldnt' this be in the MANIFEST.MF and added to our product.xml?

Reply:
I didn't add `org.apache.commons.io` into MANIFEST.MF,  because this bundle can not be resolved when jdt.ls tries to load `che.ls.jdt.ls core`, as i understand commons-io is not part of the jdt.ls runtime.

Reply:
If jdt.ls does not contain this bundle, and we're not contributing it to jdt.ls when we extend jdt.ls in Theia, where is the bundle going to come from when we run inside theia? Sorry, but I can't let this go, need to understand how this is going to work.

Reply:
@tsmaeder I've added commons-io as a library to the bundle which we're going to  install into jdt.ls.
Please take a look at this variant.